driver/src/contract.rs

```diff
@@ -70,6 +70,13 @@ impl SnappContractImpl {
 }
 
 impl SnappContract for SnappContractImpl {
+    fn get_current_block_number(&self) -> Result<U256> {
```

Comment:
Do you have any idea on what basis this ordering is happening?

Reply:
I am using `intellij-rust`

I am also not able to find this trait ordering easily, but will keep looking.